Dockerfile

```diff
@@ -96,6 +96,13 @@ RUN  mkdir -p /opt/pega/decompressedconfig  && \
      chmod -R g+rw /opt/pega/decompressedconfig && \
      chown -R pegauser /opt/pega/decompressedconfig
 
+# Create directory for deployment utilities
+RUN  mkdir -p /opt/pega/utility && \
+     chgrp -R 0 /opt/pega/utility && \
+     chmod g+rw /opt/pega/utility && \
+     chown -R pegauser /opt/pega/utility
+
+COPY versionchecker/build/libs/versionchecker-*.jar /opt/pega/utility/versionchecker.jar
```

Comment:
This is _probably_ fine, but could cause a problem if one of the alternate jars (like sources) gets enabled.